games/godot/Portfile

```diff
@@ -25,6 +25,7 @@ long_description    Godot Engine is a cross-platform game engine for \
 if {$subport eq ${name}} {
     github.setup    godotengine ${name} 3.3 "" -stable
     version         3.3.0
+    revision        1
```

Comment:
if this only simplifies the `clang` version checking it shouldn't change anything on the actual install. In that case you should not increase the revision as the installed files do not change.

Reply:
Oops, yes you're right, good catch. Fixed.